doc/appendices/command-line/traffic_layout.en.rst

```diff
@@ -103,6 +103,10 @@ Options
 
     Specify the way of copying executables when creating runroot
 
+.. option:: --layout=[<path>]
+
+    Use specific layout (providing yaml file) to create runroot
```

Comment:
YAML file is `path`? Is `path` the directory or the full file name? Can it be relative and if so, relative to what?